<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Título :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_title"/>
     </item>
     <item>
      <widget class="QCheckBox" name="check_scale">
       <property name="text">
        <string>Escala</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_legend">
       <property name="text">
        <string>Leyenda</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QMdiArea" name="mdiArea">
     <property name="viewMode">
      <enum>QMdiArea::TabbedView</enum>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsMovable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="win_ts">
      <property name="windowTitle">
       <string>Subwindow</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="checkbox_ts">
           <property name="text">
            <string>Realizar mapa</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Temático   :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_ts"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="win_vt">
      <property name="windowTitle">
       <string>Subwindow</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkbox_vt">
           <property name="text">
            <string>Realizar mapa</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Temático 1 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_comp_t1"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11"/>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Temático 2 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_comp_t2"/>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
      <zorder></zorder>
     </widget>
     <widget class="QWidget" name="win_norm">
      <property name="windowTitle">
       <string>Subwindow</string>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="checkbox_norm">
           <property name="text">
            <string>Realizar mapa</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Campo 1 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_norm_c1"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Campo 2 :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_norm_c2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_do">
       <property name="text">
        <string>Imprimir mapas (0)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
